src/reducer/TermItReducers.ts

```diff
@@ -441,7 +441,7 @@ function notifications(
 }
 
 function pendingActions(
-  state: { [key: string]: AsyncActionStatus } = {},
+  state: { [key: string]: AsyncActionStatus | AbortController } = {},
```

Comment:
I am not a fan of using two completely different types of objects as values here. Wouldn't it be in this case better to introduce a new type that would contain both the action status and the optional abort controller?